.changeset/thirty-seas-jog.md

```diff
@@ -0,0 +1,8 @@
+---
+'@twilio-paste/menu': minor
+'@twilio-paste/core': minor
+'@twilio-paste/website': patch
```

Comment:
are updating docs site examples considered a patch?

Reply:
no need to include the website changes in the changeset